src/Controller/RentalController.java

```diff
@@ -0,0 +1,268 @@
+        artworkControllerInstance.showArtworks();
+        System.out.println("Select Artwork: ");
+        String artworkName = scanner.next();
+        ArtWork artWork = artworkControllerInstance.getArtworkList().get(artworkName.toLowerCase());
+        showInstitutes();
+        System.out.println("Select Institute: ");
+        String artworkLocation = scanner.next();
+        if (instituteList.containsKey(artworkLocation.toLowerCase())) {
+            if (expositionControllerInstance.getExpoList().containsKey(artWork.getInventoryLocation())) {
+                System.out.println(artworkName + " is not available for rental \n Location: " + artWork.getInventoryLocation());
+                if ((expositionControllerInstance.onActiveExpo(artWork))) {
+                    System.out.println("Artwork can not be sent for rental. Check exposition status");
+                }
+            } else {
+                artWork.setInventoryLocation(artworkLocation);
+                System.out.println(artworkName + " will be rented by " + artWork.getInventoryLocation());
+            }
+
+            boolean done = false;
+            while (!done) {
+                System.out.println("Set rent price. (No cents)");
+                int artRentPrice = scanner.nextInt();
+                if (artRentPrice > 0) {
+                    instituteList.get(artWork.getInventoryLocation()).setRentalPrice(artRentPrice);
+                    instituteList.get(artWork.getInventoryLocation()).setPendingRental(artRentPrice);
+                    instituteList.get(artWork.getInventoryLocation()).setRentalStatus("unpaid");
+                    done = true;
+                } else {
+                    System.out.println("Error price can not be 0 or lower. \n Try again. ");
+                }
+            }
+        }
+    }
+
+    /**
+     * Method to determine an institution rent status
+     *
+     * @param artWork Artwork to retrieve location of the Institute
+     * @return The boolean flag of an Artwork rent status
+     */
+    public boolean onActiveRent(ArtWork artWork) {
+        String test = artWork.getInventoryLocation();
+        if (!instituteList.containsKey(test)) {
+            return false;
+        }
+        Rental rental = instituteList.get(test);
+        if (rental.getRentalStatus().equalsIgnoreCase("unpaid") || rental.getRentalStatus().equalsIgnoreCase("pending")){
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Method to determine if an Artwork is being rented
+     *
+     * @see #showInstitutes()
+     */
+    public void checkRentalStatus() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Please select artwork: ");
+        LinkedHashMap<String, ArtWork> artWorkList = artworkControllerInstance.getArtworkList();
+        showInstitutes();
+        String selectedArt = scanner.next();
+        ArtWork artWork = artWorkList.get(selectedArt);
```

Comment:
shouldnt you validate the artwork is not null before proceeding? u can do that here or onactiverent